RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -211,19 +211,46 @@ void resetMaximize()
 	{
 		res = defaultMaximizeStatement();
 	}
-	foreach it in $items[hewn moon-rune spoon, makeshift garbage shirt, broken champagne bottle, snow suit]
+	
+	void exclude(item it)
+	{
+		if(res != "")
+		{
+			res += ",";
+		}
+		res += "-equip " + it;
+	}
+	
+	// don't want to equip these items automatically
+	// spoon breaks mafia, snow suit bonus drops every 5 combats so is best saved for important things.
+	foreach it in $items[hewn moon-rune spoon, snow suit]
```

Comment:
I'd also suggest excluding `sword behind inappropriate prepositions` & `staph of homophones` if not maximizing rollover adventures, just in case.

Reply:
added in followup PR #368